src/fun/builtins.hvm

```diff
@@ -42,3 +45,39 @@ Map/set map key value =
         _: (Map/node value Map/leaf Map/leaf)
       }
   }
+
+// IO Impl
+
+STRING_NIL_TAG  = 0
+STRING_CONS_TAG = 1
+
+IO_DONE_TAG       = 0
+IO_PUT_TEXT_TAG   = 1
+IO_GET_TEXT_TAG   = 2
+IO_WRITE_FILE_TAG = 3
+IO_READ_FILE_TAG  = 4
+IO_GET_TIME_TAG   = 5
+IO_SLEEP_TAG      = 6
+IO_DRAW_IMAGE_TAG = 7
+
+data IO
```

Comment:
Could you add `IO/bind` like this https://github.com/HigherOrderCO/Kindex/blob/master/Data/IO/bind.kind2 ?